unlock-app/src/hooks/useEns.ts

```diff
@@ -30,7 +30,7 @@ export const getNameOrAddressForAddress = async (
 export const getAddressForName = async (_name: string): Promise<string> => {
   try {
     const name = _name.trim()
-    const isAddress = ethers.utils.isAddress(name)
+    const isAddress = name.split('.').pop()?.toLowerCase() !== 'eth'
```

Comment:
So one thing I noticed is some emails generate an invalid address by checksum but still work for other purposes. Should we try to find a way to always get checksum proof addresses for transfers?

Reply:
I don't think we need to have checksum proof addresses.